StreamChat_v3.xcodeproj/project.pbxproj

```diff
@@ -1526,7 +1550,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_TEAM = D48F367GRM;
 				INFOPLIST_FILE = Sample_v3/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 13.0;
```

Comment:
Can we leave it to 11 and use @available checks for the iOS13+ parts? We use this app for testing the older versions, too.

Reply:
@VojtaStavik yes. I'll just need to remove Combine.

Reply:
It's ok to link against it (it's weakly linked), just put any usage of it behind the availability check.